File: php/Human.php
<?php

class Human extends Mahasiswa
{
    private $nik;
    private $nama2;
    private $jenis_kelamin2;
    

    function __construct($nik = null, $nama2 = null, $jenis_kelamin2 = null)
    {
        $this->nik = $nik;
        $this->nama2 = $nama2;
        $this->jenis_kelamin2 = $jenis_kelamin2;
    }

    function setNik($nik)
    {
        $this->nik = $nik;
    }

    function getNik()
    {
        return $this->nik;
    }

    function setNama2($nama2)
    {
        $this->nama2 = $nama2;
    }

    function getNama2()
    {
        return $this->nama2;
    }

    function setJk2($jenis_kelamin2)
    {
        $this->jenis_kelamin2 = $jenis_kelamin2;
    }

    function getJk2()
    {
        return $this->jenis_kelamin2;
    }
}

File: python/human.py
from mahasiswa import mahasiswa

class human(mahasiswa):
    __nik = ""
    __nama2 = ""
    __jk2 = ""

    def __init__(self, nik="", nama2="", jk2=""):
        self.__nik = nik
        self.__nama2 = nama2
        self.__jk2 = jk2

    def setNik(self, nik):
        self.__nik = nik

    def getNik(self):
        return self.__nik

    def setNama2(self, nama2):
        self.__nama2 = nama2

    def getNama2(self):
        return self.__nama2

    def setJk2(self, jk2):
        self.__jk2 = jk2

    def getJk2(self):
        return self.__jk2
